content/en/tracing/profiling/_index.md

```diff
@@ -129,6 +129,59 @@ The Datadog Profiler requires Python 2.7+. Memory profiling only works on Python
 [3]: /tracing/visualization/#services
 {{% /tab %}}
 
+{{% tab "Go" %}}
+
+The Datadog Profiler requires Go 1.12+. To begin profiling applications:
+
+1. Install `ddtrace` which contains both tracing and profiling:
+
+    ```shell
+    go get gopkg.in/DataDog/dd-trace-go.v1/ddtrace
+    ```
+
+     **Note**: Profiling is available in the `ddtrace` library for versions 1.23.0+.
+
+2. Set `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` in your environment variables.
+
+3. Import the [Go profiler][5] at the start of your application
+
+    ```Go
+    import "gopkg.in/DataDog/dd-trace-go.v1/profiler"
+    ```
+
+4. To profile your code, add a [Datadog API key][1], then start the profiler:
+
+    ```Go
+    err := profiler.Start(
+        profiler.WithAPIKey("123key")
+    )
+    if err != nil {
+        log.Fatal(err)
+    }
+    defer profiler.Stop()
+    ```
+
+5. After a minute or two, visualize your profiles on the [Datadog APM > Profiling page][2].
+
+**Note**:
+
+- For advanced setup of the profiler or to add tags like `service` or `version`, use environment variables or `profiler` methods to set the parameters:
```

Comment:
```suggestion
Profiler configuration:
```

1. A single item list doesn't need to be a list :-)
2. It's funny to describe service and version tags as advanced since we (correctly) show them in the quickstart above.